.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+#### Issue
+* PP-XXXX
+
+#### Description
+* A few sentences describing the overall goals of the pull request's commits.
+
+#### Steps to Test or Reproduce
```

Comment:
Yes @mike0042 this information supposed to be present in JIRA ticket. I don't think QA will need to look into both ticket and pull.. QA will look only into JIRA ticket... And this information in pull is for reviewers and collaborators...
No this information will not be added to JIRA automatically.... user has to add manually